frontend/src/views/SettingsView.vue

```diff
@@ -1,7 +1,116 @@
 <template>
 	<div class="page_box_wrapper">
 		<div class="page_box">
-			<h1>This is a profile settings page</h1>
+			<h1>SETTINGS</h1>
+			<ProfileSettingAvatar :profile_picture="this.user.avatar" />
+			<div class="c_block c_form_group tac">
+				<ChangeAvatar />
+			</div>
+			<form
+				method="post"
+				action=""
+				class="c_block c_form_group"
+				@submit.prevent="
+					userStore.updateProfile(user.id, updateProfileForm)
+				"
+			>
+				<InputField
+					v-if="this.user.intra_login"
+					label="intra name"
+					v-model="this.user.intra_login"
+					is_disabled="true"
```

Comment:
should be:
``` typescript
:is_disabled="true"
```

Reply:
otherwise it's a string literal value "true", and with the colon in front it becomes the js boolean value ;-)